unittests/test_rest_framework.py

```diff
@@ -1699,6 +1699,15 @@ def __init__(self, *args, **kwargs):
         self.deleted_objects = 25
         BaseClass.RESTEndpointTest.__init__(self, *args, **kwargs)
 
+    def test_create(self):
+        payload = self.payload.copy() | {
+            "password": "testTEST1234!@#$",
+        }
+        length = self.endpoint_model.objects.count()
+        response = self.client.post(self.url, payload)
+        self.assertEqual(201, response.status_code, response.content[:1000])
+        self.assertEqual(self.endpoint_model.objects.count(), length + 1)
+
     def test_create_user_with_non_configuration_permissions(self):
```

Comment:
I guess this will fail with both `object does not exist` and a message about the user missing a password, but seems like we should supply the password just to make it clearer what's *supposed* to fail here.

Reply:
sure thing